requirements-dev.txt

```diff
@@ -1,2 +1,5 @@
 -r requirements-test.txt
 bumpversion
+isort==4.2.15
```

Comment:
IMO it's important to always have the latest version for these packages and the version pin is unnecessary. can you remove it?

Reply:
@lamenezes Yes, but I do not think it's a good idea, because for this release, isort works well with this release. I'll remove it!